core/domain/feedback_jobs_one_off.py

```diff
@@ -212,6 +212,7 @@ def map(item):
                 message_count=item.message_count,
                 last_updated=item.last_updated, created_on=item.created_on,
                 deleted=item.deleted).put(update_last_updated_time=False)
+            yield('GeneralFeedbackThreadModel', 1)
```

Comment:
See other code: there should be a space between "yield" and "(". Ditto below.

Reply:
Done